src/Analysis/Ast/Impl/Modules/Resolution/MainModuleResolution.cs

```diff
@@ -190,6 +192,19 @@ public IPythonModule GetSpecializedModule(string name)
             foreach (var s in _userPaths) {
                 _log?.Log(TraceEventType.Information, $"    {s}");
             }
+
+            _log?.Log(TraceEventType.Information, "Aliased search paths:");
+            var zipRoot = CacheFolders.GetZipCacheRoot(_services.GetService<IOSPlatform>());
+            foreach (var s in zipPaths) {
+                // Alias ../test.zip => ../test folder
+                var aliasedPath = Path.Combine(zipRoot, Path.GetFileNameWithoutExtension(s.Path));
+                if(!Directory.Exists(aliasedPath)) {
+                    _fs.ExtractZip(s.Path, aliasedPath);
+                }
```

Comment:
How would that work when zips have same names in different modules?

Reply:
Good point